mesh/mesh_connections.hpp

```diff
@@ -0,0 +1,276 @@
+// Copyright (c) 2010-2021, Lawrence Livermore National Security, LLC. Produced
+// at the Lawrence Livermore National Laboratory. All Rights reserved. See files
+// LICENSE and NOTICE for details. LLNL-CODE-806117.
+//
+// This file is part of the MFEM library. For more information and source code
+// availability visit https://mfem.org.
+//
+// MFEM is free software; you can redistribute it and/or modify it under the
+// terms of the BSD-3 license. We welcome feedback and contributions, see file
+// CONTRIBUTING.md for details.
+
+#ifndef MFEM_MESH_CONNECTIONS
+#define MFEM_MESH_CONNECTIONS
+
+#include "../general/array.hpp"
+#include <vector>
+
+namespace mfem
+{
+
+class Table;
+class Mesh;
+
+/** @brief Class that exposes connections between Mesh entities such as
+    vertices, edges, faces, and volumes.  This is done without assuming 
+    the elements have a particular dimension or type and will support 
+    dimensions > 3 in the future.  EntityIndex numbers are described in the 
+    \ref EntityIndex and \ref EntityIndices sturcts.  The connections are defined 
+    on the Mesh object given at the time of construction, and Table objects are built 
+    lazily as they are required so the first query of a paticular kind may be a lot slower
+    than the following queries.  It is worth noting that the our naming convention of child
```

Comment:
We need to choose between "the" and "our":
```suggestion
    than the following queries.  It is worth noting that our naming convention of child
```